### PR TITLE
Ddcnl 5000

### PR DESCRIPTION
**Pull request by richardjamesbaconHMRC**

### CODE REVIEW

File: app/models/HowEasyQuestion.scala [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    override def reads(json: JsValue): JsResult[HowEasyQuestion] = json match {
+      case JsString(VeryEasy.toString)      => JsSuccess(VeryEasy)
+      case JsString(Easy.toString)          => JsSuccess(Easy)
+      case JsString(Moderate.toString)      => JsSuccess(Moderate)
+      case JsString(Difficult.toString)     => JsSuccess(Difficult)
+      case JsString(VeryDifficult.toString) => JsSuccess(VeryDifficult)
+      case _                                => JsError("Unknown howEasyQuestion")
+    }
+  }
+}
```
Comment (OktawianC) [inline app/models/HowEasyQuestion.scala]: please insert new line here

File: test/controllers/ThankYouControllerSpec.scala [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    "return OK and the correct view for a GET" in {
+      val result = controller().onPageLoad(fakeRequest)
+
+      status(result) mustBe OK
+      contentAsString(result) mustBe viewAsString()
+    }
+  }
+}
+
+
```
Comment (OktawianC) [inline test/controllers/ThankYouControllerSpec.scala]: please delete lines from 41 to 43

File: app/views/govuk_wrapper.scala.html [hunk 3]
```diff
@@ -30,6 +30,11 @@
 @import uk.gov.hmrc.play.views.html.{helpers => uiHelpers}
 
 @head = {
+
+    @if(appConfig.isGtmEnabled) {
+      @snippets.google_tag_manager(appConfig)
+    } else {}
```
Comment (gorkemunsal) [inline app/views/govuk_wrapper.scala.html]: do you need the else clause here it doesnthave anything

File: app/views/govuk_wrapper.scala.html [hunk 4]
```diff
@@ -47,7 +52,11 @@
       navLinks = Some(headerNavLinks))
 }
 
-@afterHeader = {}
+@afterHeader = {
+    @if(appConfig.isGtmEnabled) {
+      @snippets.google_tag_manager_noscript(appConfig)
+    } else {}
```
Comment (gorkemunsal) [inline app/views/govuk_wrapper.scala.html]: same as above else statement doesnt do anything

**Pull request merged**